src/main/java/org/apache/commons/lang3/ComparableUtils.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.lang3;
+
+/**
+ * <p>Utility library to provide helper methods for translating {@link Comparable#compareTo} result into a boolean.</p>
+ *
+ * <p>Example: {@code boolean x = is(myComparable).lessThanOrEqualTo(otherComparable)}</p>
+ *
+ * <p>#ThreadSafe#</p>
+ *
+ * @since 3.8.2
+ */
+public class ComparableUtils {
+
+    /**
+     * Provides access to the available methods
+     *
+     * @param a   base object in the further comparison
+     * @param <T> type of the base object
+     * @return a builder object with further methods
+     */
+    public static <T extends Comparable<T>> ComparableCheckBuilder<T> is(T a) {
+        return new ComparableCheckBuilder<>(a);
+    }
+
+    /**
+     * Provides access to the available methods
+     */
+    public static class ComparableCheckBuilder<A extends Comparable<A>> {
+
+        private final A a;
+
+        private ComparableCheckBuilder(A a) {
+            this.a = a;
+        }
+
+        /**
+         * Checks is the value returned by {@link Comparable#compareTo} is equal to {@code 0}
```

Comment:
I think all the javadocs here have the same typo, "Checks is the value..." should be "Check if the value..." ?

Reply:
It should read "Checks if the value..."

Reply:
Ops, good spot @garydgregory

Reply:
fixed thanks